src/applications/personalization/profile/tests/components/contact-information/SignInServiceUpdateLink.unit.spec.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { expect } from 'chai';
+import SignInServiceUpdateLink from '../../../components/contact-information/email-addresses/SignInServiceUpdateLink';
+import { renderInReduxProvider } from '~/platform/testing/unit/react-testing-library-helpers';
+import { SERVICE_PROVIDERS } from '~/platform/user/authentication/constants';
+
+Object.values(SERVICE_PROVIDERS).forEach(csp => {
```

Comment:
Credential Service Provider (see page 48 https://www.va.gov/files/2022-12/VA%20Acronym%20Lookup.pdf)

I was kinda confused too, because I was thinking of Content Security Policy!